Readability.js

```diff
@@ -1806,6 +1815,9 @@ Readability.prototype = {
         metadata.excerpt = paragraphs[0].textContent.trim();
       }
     }
+    if (this._isRtl(articleTitle)){
```

Comment:
This doesn't do what you said in the description of the PR - it always overrides the article direction, even if the direction *was* specified. Please check whether `_articleDir` is specified.